src/elements/content-sidebar/activity-feed/task-form/__tests__/__snapshots__/TaskError.test.js.snap

```diff
@@ -10,36 +10,7 @@ exports[`components/content-sidebar/activity-feed/task-form/TaskError when type
   }
   taskType="APPROVAL"
 >
-  <InlineNotice
-    title={
-      <FormattedMessage
-        defaultMessage="Error"
-        id="be.contentSidebar.activityFeed.taskForm.taskCreateErrorTitle"
-      />
-    }
-    type="error"
-  >
-    <div
-      className="inline-alert inline-alert-visible inline-alert-error"
-    >
-      <strong>
-        <FormattedMessage
-          defaultMessage="Error"
-          id="be.contentSidebar.activityFeed.taskForm.taskCreateErrorTitle"
-        >
-          <div />
-        </FormattedMessage>
-      </strong>
-      <div>
-        <FormattedMessage
-          defaultMessage="An error occurred while creating this task. Please try again."
-          id="be.api.taskCreateErrorMessage"
-        >
-          <div />
-        </FormattedMessage>
-      </div>
-    </div>
-  </InlineNotice>
+  <div />
```

Comment:
There should be one new snapshot, but many of them have been replaced with empty containers.